### ISSUE TITLE
paypal/rest-api-sdk-php version

### ISSUE THREAD

**Issue by OtiZ2** [anouarabdsslm/laravel-paypalpayment#161]
Hi i need use 
paypal/rest-api-sdk-php 1.14.0

It posibble to update?

**Comment by mmejaz** [comment]
please let me know, how can this package work for updated PayPal API

**Comment by anouarabdsslm** [comment]
Guys , I was maintaining this package for a while for tremendous reasons, I will update the package this weekend to use the latest versions of laravel.

**Comment by mmejaz** [comment]
very nice anouarabdsslm. and secondly please manage it with the latest PayPal SDK (Installing paypal/rest-api-sdk-php (v0.16.1)) instead of the old.
it will be considered a great favor from your die.
thanks

**Comment by mmejaz** [comment]
i am waiting

**Comment by anouarabdsslm** [comment]
yes I found out that paypal SDK is deprecated https://github.com/paypal/PayPal-PHP-SDK I will try to check their new packages to see how I can re-implement it with the package. I will let you know when the package is ready.

**Comment by mmejaz** [comment]
good, let me know are you working on it. and when you will push it to GitHub.
